### PR TITLE
fix: update Navigation tab items styling to match design.

### PR DESCRIPTION
**Pull request by saadyousafarbi**
### Description
Styling fix for Navigation tab items for the hover state.

#### Hover for inactive tab
Screen Shot 2021-07-14 at 1 03 35 AM

#### Hover for active tab
Screen Shot 2021-07-14 at 1 03 51 AM

#### Focus state
Screen Shot 2021-07-27 at 1 19 41 AM

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -70,6 +70,19 @@
...
+
+  .nav a, .nav button {
+    &:hover {
+      background-color: $gray-400;
+    }
+  }
+
+  .nav a {
+    &:not(.active):hover {
+      background-color: $gray-400;
```
Comment (marcotuts) [inline src/index.scss]: I think this should switch to $light-400 (it's a lighter gray color)

File: src/index.scss [hunk 2]
```diff
@@ -70,6 +70,19 @@
 }
 .course-tabs-navigation {
   border-bottom: solid 1px #EAEAEA;
+
+  .nav a, .nav button {
+    &:hover {
+      background-color: $gray-400;
```
Comment (marcotuts) [inline src/index.scss]: Same note here -

I think this should switch to $light-400 (it's a lighter gray color)

**Review by marcotuts — CHANGES_REQUESTED** [review]
Not sure if comments are visible until I create a review (mobile UI on GitHub makes it seem like my comments are still pending.) 

Added two online comments

**Comment by saadyousafarbi** [comment]
@marcotuts updated PR and screenshot with updated color! Please have a look, Thank you!

**Review by awais-ansari — COMMENTED** [review]
Everything is good. Just need to add a focus effect on tabs. 
https://www.figma.com/file/eGmDp94FlqEr4iSqy1Uc1K/Paragon-Design-System?node-id=9076%3A672

**Comment by saadyousafarbi** [comment]
@awais-ansari thanks for pointing that out. Have added `focus` state, and added a screenshot within the PR description.

**Review by awais-ansari — APPROVED** [review]

**Review by marcotuts — APPROVED** [review]

**Pull request merged**